client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEditForm.tsx

```diff
@@ -73,7 +73,8 @@ export const PrescriptionLineEditForm: React.FC<
   const [allocationAlerts, setAllocationAlerts] = useState<StockOutAlert[]>([]);
   const [issueQuantity, setIssueQuantity] = useState(0);
   const { format } = useFormatNumber();
-  const { items } = usePrescription.line.rows();
+  // const { items } = usePrescription.line.rows();
+  const { rows: items } = usePrescription();
```

Comment:
There was a lot of this kind of thing, where there was a whole hook for constructing data structures that can be easily derived from the main prescription data, so I've rolled them in to the main prescription hook.